ansible/pbTestScripts/README.md

```diff
@@ -12,16 +12,19 @@ This folder contains the scripts necessary to start separate vagrant machines wi
 * Ubuntu 16.04
 * Ubuntu 18.04
 * Ubuntu 20.04
+* Ubuntu 21.04
+* Ubuntu 22.04
+* Ubuntu 24.04
 * CentOS6
 * CentOS7
 * CentOS8
-* Debian8
-* Debian10
-* FreeBSD12
+* Debian11
+* Debian12
 * Solaris10
 * Windows Server 2012 R2
+* Windows Server 2022
```

Comment:
Windows Server 2025 incoming?

Reply:
Issue : https://github.com/adoptium/infrastructure/issues/4031 raised to look at adding support, probably dependent on the windows 2025 test commisioning currently planned for Iteration 19 of the Adoptium project plan.